grafana-plugin/src/containers/ScheduleSlot/ScheduleSlot.module.css

```diff
@@ -56,9 +56,8 @@
   z-index: 1;
   color: #fff;
   font-size: 12px;
+  width: 100%;
   font-weight: 500;
-  pointer-events: none;
```

Comment:
This one blocked showing tooltip within schedule slot, it was added long time ago and as far as I tested, everything works without it. But if you remember some context about it @maskin25 , pls let me know

Reply:
hmm, I haven't seen any issues with tooltip over schedule slot, and tooltip is attached to a higher level container, not to the title element. I forgot the reason of adding `pointer-events: none` here, I'm almost sure there was some reason but probably it doesn't make sense anymore  after several iterations with schedules and visual components rearranging

Reply:
Got it, thanks!